t/make_executable.t

```diff
@@ -2,32 +2,50 @@
 
 use strict;
 use warnings FATAL => 'all';
+use English;
 
 use Config;
 use Test::More;
 use ExtUtils::PL2Bat;
 use Cwd qw/cwd/;
 
-plan($^O eq 'MSWin32' ? (tests => 7) : skip_all => 'Only usable on Windows');
+use IPC::System::Simple qw/ capture system EXIT_ANY /;
```

Comment:
As this distribution is intended to be included in core, non-core dependencies are not appropraite.